docs: add tutorials for running macaron (#232)
oracle-samples/macaron: docs/source/pages/installation.rst

.. Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.

.. _installation-guide:

============

> Installation

Reviewer: ```suggestion
Installation Guide
```

Author: Fixed in https://github.com/oracle-samples/macaron/pull/232/commits/11976b13ad452d9a8c94e09cecb9aeca5c34e004

============

-------------
Prerequisites
-------------
- Installations of ``wget`` or ``curl`` and ``bash`` must be available and on the path.

> - Docker (or docker equivalent for your host OS) must be installed, with a docker command line equivalent to Docker 17.06 (Oracle Container Runtime 19.03) and the user that is executing a member of the operating system group ``docker`` (to run Docker in `rootless mode <https://docs.docker.com/engine/security/rootless/>`_).

Reviewer: ```suggestion
- Docker (or docker equivalent for your host OS) must be installed, with a docker command line equivalent to Docker 17.06 (Oracle Container Runtime 19.03) and the user should be a member of the operating system group ``docker`` (to run Docker in `rootless mode <https://docs.docker.com/engine/security/rootless/>`_).
```

Author: Fixed in https://github.com/oracle-samples/macaron/pull/232/commits/11976b13ad452d9a8c94e09cecb9aeca5c34e004
